easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -107,7 +109,8 @@ def configure_step(self, srcdir=None, builddir=None):
 
         options_string = ' '.join(options)
 
-        command = "%s cmake %s %s %s" % (self.cfg['preconfigopts'], srcdir, options_string, self.cfg['configopts'])
+        command = "%s %s %s %s %s" % (self.cfg['preconfigopts'], self.cfg['configure_cmd'], srcdir, options_string,
+                                      self.cfg['configopts'])
```

Comment:
@mboisson Do you mind changing this to:

```python
command = ' '.join([
    self.cfg['preconfigopts'],
    self.cfg['configure_cmd'],
    srcdir,
    options_string,
    self.cfg['configopts'],
])
```

(similar below, but you can make it a single line if it's short enough)